position_to_mavros:
  only_vision: True
  pose_pub_topic: "/uav/mavros/vision_pose/pose"
  uwb_sub_topic: "/dwm1001/tag/dronie/position"
  vision_sub_topic: "/uav/camera/odom/sample"
  lidar_sub_topic: "/uav/tfmini_ros_node/range"
  
safe_offboard:
  fly_time: 40
  flight_mode: "stay"
  takeoff_height: 1.0
  circle_radius: 0.6
  pos_valid_time: 1.0
  waypoint_valid_time: 1.0
  fly_fence: [-1.5, -1.5, 0.0, 1.5, 1.5, 1.5]


## TODO: Add more params needed
# offboard_node: